crates/toml/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "toml"
-version = "0.9.9+spec-1.0.0"
+version = "0.9.9+spec-1.1.0"
```

Comment:
Blocked on the release of TOML 1.1: toml-lang/toml#928

Reply:
#623 is the main candidate for `toml_edit` (the only package with a breaking change).  Others will require more investigation and more work to update to it.

Reply:
just in case: those two versions numbers are equivalent and you won't be able to publish them aside each other